test/Microsoft.Azure.SignalR.AspNet.Tests/E2ETest/TestHub.cs

```diff
@@ -1,15 +1,96 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Web.UI.WebControls;
 using Microsoft.AspNet.SignalR;
+using Serilog;
+using Serilog.Core;
 
 namespace Microsoft.Azure.SignalR.AspNet.Tests
 {
     public class TestHub : Hub
     {
+        private static ConcurrentDictionary<string, bool> _connectedConnections = new ConcurrentDictionary<string, bool>();
```

Comment:
>static [](start = 16, length = 6)

why `static`?

Reply:
When you call a method in hub, the hub is created by reflection, so the non-static member is created and disposed for every call/request. So I need to kepp it static to use the data across methods.

Reply:
How about using a `ConnectionManager` instead?